easybuild/easyconfigs/b/Biopython/Biopython-1.72-foss-2018b-Python-2.7.15.eb

```diff
@@ -31,11 +33,10 @@ dependencies = [
 
 download_dep_fail = True
 
-use_pip = True
```

Comment:
@fizwit You're modifying an existing easyconfig file here, which was contributed via #6925 .

I don't mind the style changes in `description` and `sanity_check_paths`, but removing the `use_pip = True` should not be done, please fix?